recipes/ckanweb-deploy.rb

```diff
@@ -322,3 +336,11 @@
 	group "root"
 	mode "0755"
 end
+
+# only have one server trigger harvest initiation, which then worker queues havester fetch/gather works through the queues.
+file "/etc/cron.hourly/ckan-harvest-run" do
+	content "/usr/local/bin/pick-job-server.sh && #{paster} --plugin=ckanext-harvest harvester run -c #{config_file} 2>&1 > /dev/null\n"
```

Comment:
I was under the impression that we don't need pick-job-server here.

Reply:
the cron  job is to kick off once a day, it should have 3 try at the same time, this job adds to the queues which the supervisord processes go through. see https://confluence.csiro.au/display/seegrid/CKAN+Harvesting+User+Guide